src/main/resources/application.properties

```diff
@@ -124,3 +124,4 @@ usermetrics.metricsReportingPoolSize=10
 usermetrics.syncRefreshIntervalSeconds=3600
 usermetrics.ignorePaths=/api/repository/v1/jobs/*,/api/repository/v1/configs*,/api/repository/v1/upgrade,/actuator/*,/configuration,/status
 rawls.basePath=https://rawls.dsde-dev.broadinstitute.org
+duos.basePath=https://consent.dsde-dev.broadinstitute.org
```

Comment:
We will need to set `DUOS_BASEPATH` environment variable overrides for the different consent envs:
https://broadworkbench.atlassian.net/browse/DR-2825

Reply:
Thanks for the heads up!  For the time being there are no integration tests but this is likely to change.